### DISCUSSION TITLE
Alpine is not defined when disable 'show_process_bar' in config file

### DISCUSSION THREAD

**Discussion by abrahamgreyson** [livewire/livewire#6548] [7. Bugs (archived)]
### Livewire version

3.0.1

### Which PHP version are you using?

PHP 8.2

### Steps To Reproduce

1. `composer require filament/filament:"^3.0"` this will install Livewire 3
2. `pa filament:install --panels` to publish Filament assets 
3. `composer info livewire/livewire` shows versions : * v3.0.1
4. `pa vendor:publish --tag=livewire` publish the config file
5. `vim config/livewire.php`  set `navigate.show_progress_bar` to `false`
6. Visit Filament Admin Panel page (/admin),   open the console,  shows 'Uncaught ReferenceError: Alpine is not defined
    at livewire.js?id=11c49d7e:6876:30'

The code with error is :
```javascript
  // js/features/supportNavigate.js
  var isNavigating = false;
  shouldHideProgressBar() && Alpine.navigate.disableProgressBar();
```

### Additional information

When "show_progress_bar" is set to false, the <script src="/livewire/livewire.js?id=11c49d7e"> code has an extra attribute "data-no-progress-bar".

It may related with https://github.com/livewire/livewire/discussions/6318

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [ ] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by vinnydude** [comment]
I'm getting this issue as well when I set "show_progress_bar" to false in the config.

**Comment by lrljoe** [comment]
#7158 <- this PR may fix your issue.

**Comment by joshhanley** [comment]
We're in the process of investigating all wire:navigate issues and I have managed to replicate this scenario and have submitted a fix in PR #9379. Thanks!
